library/src/main/java/org/angmarch/views/NiceSpinner.java

```diff
@@ -30,7 +31,6 @@
 import android.widget.ListPopupWindow;
 import android.widget.ListView;
 
-import java.util.ArrayList;
```

Comment:
:thinking:  Was this part of an "Optimize imports" action? are there more unused imports in this file?